examples/src/main/java/org/apache/spark/examples/sql/JavaSQLDataSourceExample.java

```diff
@@ -106,6 +107,43 @@ public static void main(String[] args) {
     spark.stop();
   }
 
+  private static void runGenericFileSourceOptionsExample(SparkSession spark) {
+    // $example on:ignore_corrupt_files$
+    // enable ignore corrupt files
+    spark.sql("set spark.sql.files.ignoreCorruptFiles=true");
+    // dir1/file3.json is corrupt from parquet's view
+    Dataset<Row> testCorruptDF = spark.read().parquet(
+            "examples/src/main/resources/dir1/",
+            "examples/src/main/resources/dir1/dir2/");
+    testCorruptDF.show();
+    // +-------------+
+    // |         file|
+    // +-------------+
+    // |file1.parquet|
+    // |file2.parquet|
+    // +-------------+
+    // $example off:ignore_corrupt_files$
+    spark.sql("set spark.sql.files.ignoreMissingFiles=false");
```

Comment:
Why do we need this

Reply:
turn off the feature to not affect the following examples that test other features.

Reply:
I think it is confusing here. The conf `spark.sql.files.ignoreMissingFiles` is disabled by default.

Reply:
I guess setting the config explicitly here is for corresponding with the doc: https://github.com/apache/spark/pull/27415/files#diff-dab20ee598064a54fecef8842b972944R72